src/dxtbx/nexus/__init__.py

```diff
@@ -224,7 +224,7 @@ def get_dxtbx_scan(
                 "collection."
             )
 
-        oscillation = (start.magnitude, step.magnitude)
+        oscillation = tuple(float(f) for f in (start.magnitude, step.magnitude))
```

Comment:
Seems possibly out of context for this PR? This is a `pint` thing right? Wondering why needed here not as a separate change?

Reply:
No, it was needed here as (at least in the example filewriter dataset I was testing against) it is returned as a `numpy.float` rather than a Python `float`